test/interface/static_array_tests.jl

```diff
@@ -28,14 +28,14 @@ sol = solve(ode, SSPRK22(), dt = 1e-2)
 sol = solve(ode, ROCK4())
 @test !any(iszero.(sol(1.0))) && !any(sol(1.0) .== u0)
 
-u0 = ones(MVector{2, Float64})
+u0 = VectorOfArray(ones(MVector{2, Float64}))
```

Comment:
I'm not sure this gives the desired result since it created a single `MVector` before:

```
julia> ones(SVector{2, Float64})
2-element SVector{2, Float64} with indices SOneTo(2):
 1.0
 1.0

julia> ones(MVector{2, Float64})
2-element MVector{2, Float64} with indices SOneTo(2):
 1.0
 1.0
```

Reply:
What do you mean?

Reply:
`ones(::Type{StaticArray})` creates a `StaticArray` and not a Vector of StaticArrays

Reply:
I get

```julia
julia> using Pkg; Pkg.activate(temp = true); Pkg.add(["StaticArrays", "RecursiveArrayTools"])
[...]
  [731186ca] + RecursiveArrayTools v3.0.0
  [90137ffa] + StaticArrays v1.7.0
[...]

julia> using StaticArrays, RecursiveArrayTools

julia> ones(SVector{2, Float64})
2-element SVector{2, Float64} with indices SOneTo(2):
 1.0
 1.0


julia> VectorOfArray(ones(SVector{2, Float64}))
VectorOfArray{Float64,1}:
2-element SVector{2, Float64} with indices SOneTo(2):
 1.0
 1.0
```

Thus, the previous version basically checks the implementation with arrays of type `SVector` or `MVector`. The new version performs checks with a `VectorOfArray` which basically acts like an array of `SVector`s/`MVector`s